src/main/java/org/jitsi/jicofo/SSRCValidator.java

```diff
@@ -53,135 +53,140 @@
     private final String endpointId;
 
     /**
-     * The SSRC map obtained from the participant which reflects current SSRC
+     * The source map obtained from the participant which reflects current source
      * status. It's a clone and modifications done here do not affect
      * the version held by {@link Participant}.
      */
-    private final MediaSSRCMap ssrcs;
+    private final MediaSourceMap sources;
 
     /**
-     * Same as {@link #ssrcs}, but for SSRC groups.
+     * Same as {@link #sources}, but for source groups.
      */
-    private final MediaSSRCGroupMap ssrcGroups;
+    private final MediaSourceGroupMap sourceGroups;
 
     /**
-     * The limit SSRCs count per media type allowed to be stored by
+     * The limit sources count per media type allowed to be stored by
      * the {@link Participant} at a time.
+     * NOTE(bbaldino): changing this to max source count to have it
+     * apply to rids as well, not sure if that's appropriate
```

Comment:
gonna get rid of this note, i had it as more of a TODO to check with you.  do you think it's alright to have it apply to ssrc + rids?  i don't think it makes sense to just be one or the other, but currently there will be duplication (each stream from firefox will have an ssrc and an rid souce, so there will be 2x normal...i don't know what we really rely on this for though)